assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -0,0 +1,225 @@
+import { PluginSidebar, PluginSidebarMoreMenuItem } from "@wordpress/edit-post";
+import { Component } from '@wordpress/element';
+import { PanelBody, RadioControl, SelectControl } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import ColorPaletteControl from '../ColorPaletteControl/ColorPaletteControl';
+import { withPostMeta } from '../PostMeta/withPostMeta';
+import { __ } from '@wordpress/i18n';
+import { fromThemeOptions } from '../fromThemeOptions/fromThemeOptions';
+
+const themeOptions = [
+  {
+    value: '',
+    label: 'Default',
+  },
+  {
+    value: 'antarctic',
+    label: 'Antarctic',
+  },
+  {
+    value: 'arctic',
+    label: 'Arctic',
+  },
+  {
+    value: 'climate',
+    label: 'Climate Emergency',
+  },
+  {
+    value: 'forest',
+    label: 'Forest',
+  },
+  {
+    value: 'oceans',
+    label: 'Oceans',
+  },
+  {
+    value: 'oil',
+    label: 'Oil',
+  },
+  {
+    value: 'plastic',
+    label: 'Plastics',
+  },
+
+];
+
+const ThemeSelect = withPostMeta( SelectControl );
+
+const Select = compose(
+  fromThemeOptions,
+  withPostMeta,
+)( SelectControl );
+
+const Radio = compose(
+  fromThemeOptions,
+  withPostMeta,
+)( RadioControl );
+
+const ColorPalette = compose(
+  fromThemeOptions,
+  withPostMeta,
+)( ColorPaletteControl );
+
+
+export class CampaignSidebar extends Component {
+  static getId() {
+    return 'planet4-campaign-sidebar';
+  }
+
+  static getIcon() {
+    return 'admin-appearance';
+  }
+
+  constructor( props ) {
+    super( props );
+    this.state = {
+      theme: null,
+    };
+    this.handleThemeChange = this.handleThemeChange.bind( this );
+  }
+
+  componentDidMount() {
+    const meta = wp.data.select( 'core/editor' ).getEditedPostAttribute( 'meta' );
+    if (meta) {
+      const theme = meta[ 'campaign_page_template' ];
+      this.loadTheme( theme );
+    }
+  }
+
+  handleThemeChange( value ) {
+    this.loadTheme( value );
+  }
+
+  loadTheme( value ) {
+    if ( value === '' || !value ) {
+      value = 'default';
+    }
+    const baseUrl = window.location.href.split( '/wp-admin' )[ 0 ];
+    const campaignConfigUrl = `${ baseUrl }/wp-content/themes/planet4-master-theme/campaign_templates/${ value }.json`;
+    fetch( campaignConfigUrl )
+      .then( response => response.json() )
+      .then( json => {
+        this.setState( prevState => {
+          return {
+            theme: json,
+          };
+        } );
+      } );
+  }
+
+  render() {
+    return (
+      <>
+        <PluginSidebarMoreMenuItem
+          target={ CampaignSidebar.getId() }
+          icon={ CampaignSidebar.getIcon()}>
+          Your Sidebar Plugin
```

Comment:
I guess this could be "Campaign Options", same as below.

Reply:
Maybe I'll just remove `PluginSidebarMoreMenuItem`, as I added a css fix that removes the unpin button. So if we remove this one too then the user can't unpin it in the first place.

Reply:
In the end I added it again, just in case the css fix would fail as suggested by Pablo